src/clj/game/cards-agendas.clj

```diff
@@ -33,6 +33,27 @@
                                                                              (resolve-ability state side (abt 1 n) card nil))}
                                                                card nil)))}}})
 
+   "Advanced Concept Hopper"
+   {:events
+    {:run
+     {:req (req (first-event state side :run))
+      :effect (effect (show-wait-prompt :runner "Corp to use Advanced Concept Hopper")
+                      (resolve-ability
+                        {:optional
+                         {:player :corp
+                          :prompt "Use Advanced Concept Hopper to draw 1 card or gain 1 [Credits]?" :once :per-turn
+                          :yes-ability {:prompt "Draw 1 card or gain 1 [Credits]?"
+                                        :choices ["Draw 1 card" "Gain 1 [Credits]"]
+                                        :effect (req (if (= target "Gain 1 [Credits]")
+                                                       (do (gain state :corp :credit 1)
+                                                           (system-msg state :corp
+                                                                       (str "uses Advanced Concept Hopper to gain 1 [Credits]")))
+                                                       (do (draw state :corp)
```

Comment:
The same issue as with KGRT, multiples of this card scored can get multiple prompts in confusing order. Every first run.

Although the multiple prompts do serve a purpose here, opposed to KGRT, so they need to stay - I can't think of a better solution here. Maybe providing a single prompt with options dependent on number of ACH scored, example with 2 ACH:

* Draw 2 cards
* Draw card gain 1 cred
* Gain 2 creds
* Cancel

(I'm aware that this skips the options of just drawing 1 or just gaining 1 cred).

In fact, this may not be better at all, disregard this comment ;-)